golem-service-base/src/service/auth.rs

```diff
@@ -16,6 +21,8 @@ pub enum AuthError {
     Unauthorized(String),
     #[error("Forbidden: {0}")]
     Forbidden(String),
+    #[error("Not found: {0}")]
+    NotFound(String),
```

Comment:
it is little bit worrying  that Auth Service can return error like NotFound

scenario like: it will return error like Account not found which with some details, which in my opinion is security issue

Reply:
What is the alternative? We already have similar errors everywhere in the codebase, especially in grpc types. What if you are trying to authenticate a given template id, and then that template id doesn't exist? What should AuthService return? 

The only alternative that I can think of is returning an anyhow::Error and downcasting it? Or including a new type parameter for the error type in AuthService? 

This is also an internal service, and the error messages don't necessarily need to be propagated to API level.